db/fixtures/miq_product_features.yml

```diff
@@ -5123,3 +5123,36 @@
     :description: Delete Arbitration Settings
     :feature_type: admin
     :identifier: delete_arbitration_setting
+
+# Blueprints
+- :name: Blueprints
+  :description: All Blueprints Access Rules
+  :feature_type:
```

Comment:
@h-kataria I tried to follow by example here, but can you advise on the feature types for these and possibly on the correctness of the configuration I have here so far?